tensorrt_llm/_torch/attention_backend/sparse/rocket/backend.py

```
@@ -0,0 +1,611 @@
+)
+from .metadata import RocketTrtllmAttentionMetadata, RocketVanillaAttentionMetadata
+
+if TYPE_CHECKING:
+    from tensorrt_llm.llmapi.llm_args import SparseAttentionConfig
+
+
+class RocketTrtllmAttention(TrtllmAttention):
+    Metadata = RocketTrtllmAttentionMetadata
+
+    # Access type for different dtype sizes
+    _access_type = {1: torch.int8, 2: torch.int16, 4: torch.int32, 8: torch.int64}
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)